github/pulls_comments.go

```diff
@@ -130,6 +136,27 @@ func (s *PullRequestsService) CreateComment(ctx context.Context, owner string, r
 	return c, resp, nil
 }
 
+// CreateCommentWithComfortFade creates a new comment on the specified pull request. It has `Accept: application/vnd.github.comfort-fade-preview+json` header.
```

Comment:
Let's please not make two versions of the same endpoint (one with the custom header and one without).
In this repo, we decided to always be on the cutting edge and support all preview APIs as best we can so that we can give early feedback to the GitHub API support team and to give clients using this library access to the latest GitHub features.

So let's please add the custom media header to the original endpoint and remove this new function.

Also, please take a look at `github/github.go` for how we add custom media headers.

Finally, please write unit tests for all functions that are affected by this PR.  Thanks.